Resources/ServerInfo/_NF/Guidebook/Shipyard/Akupara.xml

```diff
@@ -0,0 +1,131 @@
+<Document>
+  # AKUPARA-CLASS BOTANICAL SHUTTLE
+  <Box>
+  <GuideEntityEmbed Entity="chem_master"/>
+  <GuideEntityEmbed Entity="SeedExtractor"/>
+  <GuideEntityEmbed Entity="hydroponicsTrayAnchored"/>
+  </Box>
+  <Box>
+  <GuideEntityEmbed Entity="UnstableMutagenChemistryBottle"/>
+  <GuideEntityEmbed Entity="FoodSpacemansTrumpet"/>
+  <GuideEntityEmbed Entity="GasVentScrubber"/>
+  <GuideEntityEmbed Entity="StorageCanister"/>
+  </Box>
+  [color=#a4885c]Ship Size:[/color] Medium
+
+  [color=#a4885c]Recommended Crew:[/color] 1-3
+
+  [color=#a4885c]Power Gen Type:[/color] Uranium
+
+  [color=#a4885c]Expeditions:[/color] No
+
+  [color=#a4885c]IFF Console:[/color] None
+
+  [color=#a4885c]Available Roles:[/color] Contractor, Pilot, Mercenary
+
+  "A medium botanical research biodome equipped with a non-standard atmospherics system capable of isolating and storing - but not processing gases produced by mutated plants."
+
+  # Piloting
+  <Box>
+    <GuideEntityEmbed Entity="ComputerShuttle"/>
+  </Box>
+  After clicking the shuttle console, you should see a radar view of the shuttle. Here are the steps for piloting the shuttle back and forth:
+
+  - First, disconnect any airlocks that are connected to the dock.
+  - Then, you actually get to pilot the shuttle. The controls are fairly simple, with [color=#028ed9]"W"[/color] and [color=#028ed9]"S"[/color] being forward and backward, [color=#028ed9]"A"[/color] and [color=#028ed9]"D"[/color] being left and right, and [color=#028ed9]"Q"[/color] and [color=#028ed9]"E"[/color] being rotating left and right; and [color=#028ed9]"spacebar"[/color] being the brake and moving precicely by holding the [color=#028ed9]"spacebar"[/color] while doing other inputs.
+  
+  # Hiring crew
+  <Box>
+    <GuideEntityEmbed Entity="ComputerStationRecords"/>
+    <GuideEntityEmbed Entity="ClothingHeadsetGrey"/>
+    <GuideEntityEmbed Entity="IntercomCommon"/>
+  </Box>
+  As a Captain of a Nanotrasen vessel, you have the authority to hire, fire, demote, or promote crew members at will. There are two ways you as a Captain can go about hiring crew: you can either use the [color=#a4885c]station records computer[/color] on your ship to open crew slots (jobs available vary from ship to ship) or invite through in-game communications other players to your crew.
```

Comment:
Sections for `Piloting` and `Hiring crew` no longer needed - those were moved to a separate guidebook entries

Reply:
Guide redundancy removed.